packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -258,34 +259,33 @@ export const QueryBar: React.FunctionComponent<QueryBarProps> = ({
           aria-label="Reset query"
           data-testid="query-bar-reset-filter-button"
           onClick={onReset}
-          disabled={!queryChanged}
+          disabled={!queryChanged || isAIFetching}
           size="small"
           type="button"
         >
           Reset
         </Button>
         <Button
           data-testid="query-bar-apply-filter-button"
-          disabled={!isQueryValid}
+          disabled={!isQueryValid || isAIFetching}
           variant="primary"
           size="small"
           type="submit"
           onClick={onFormSubmit}
         >
           {buttonLabel}
         </Button>
-        {showExportToLanguageButton && (
```

Comment:
Looks like it's used here? If I remember correctly this is used for cloud to hide the export to language functionality as it isn't supported.

Reply:
Ah cloud sets it! Got it, then I will revert this change. Thx for bringing this up!